### PR TITLE
Hallucinations 2.2

### PR DESCRIPTION
**Pull request by XDTM**
Here we go again

The door electrocution now looks realistic instead of being message + stun. Variance on sounds is defaulted to 0 on playsound_local, so i had to define it manually on the procs.

### CODE REVIEW

**Comment by jughu** [comment]
You taste skub!

File: code/modules/mob/living/taste.dm [hunk 1]
```diff
@@ -21,6 +21,8 @@
 		var/text_output = from.generate_taste_message(taste_sensitivity)
 		// We dont want to spam the same message over and over again at the
 		// person. Give it a bit of a buffer.
+		if(hallucination > 50 && prob(25))
+			text_output = pick("spiders","[pick(wire_colors)]","dreams","nightmares","the future","the past","victory","defeat","pain","bliss","revenge","poison","time","space","death","life")
```
Comment (Cheridan) [inline code/modules/mob/living/taste.dm]: snozzberries

Comment (ChangelingRain) [inline code/modules/mob/living/taste.dm]: truth
justice
memory
your soul
suffering
music
blood

Comment (Cheridan) [inline code/modules/mob/living/taste.dm]: the american way
also add everything listed here in this thread: https://tgstation13.org/phpBB/viewtopic.php?f=55&t=9256

**Pull request merged**